src/app/collective-rewards/metrics/CycleMetrics.tsx

```diff
@@ -0,0 +1,48 @@
+import { MetricsCard } from '@/components/MetricsCard'
+import { useCycleContext } from '@/app/collective-rewards/metrics'
+import { Duration, DateTime } from 'luxon'
+import { useEffect, useState } from 'react'
+
+let timeout: NodeJS.Timeout
+
+export const CycleMetrics = () => {
+  const [timeRemaining, setTimeRemaining] = useState<Duration>(Duration.fromObject({ minutes: 0 }))
+  let {
+    data: { cycleDuration, cycleNext },
+  } = useCycleContext()
+
+  const duration =
+    cycleDuration.as('days') < 1 ? cycleDuration.shiftTo('hours') : cycleDuration.shiftTo('days')
+
+  useEffect(() => {
+    if (timeRemaining.as('minutes') > 0) {
+      timeout = setTimeout(() => {
+        setTimeRemaining(state => state.minus({ minutes: 1 }))
+      }, 60000) // every minute
+    } else {
+      clearTimeout(timeout)
+    }
+
+    return () => clearTimeout(timeout)
+  }, [timeRemaining.minutes])
```

Comment:
Although it's clear to me, let's add eslint exception here to clarify the reasoning